lib/utils/isDynamoObject.js

```diff
@@ -0,0 +1,9 @@
+const AWS = require('aws-sdk');
+
+module.exports = (item) => {
+  if (AWS.DynamoDB.Converter.output(item)) {
```

Comment:
why not just return this check? An `if else` that returns a `Boolean` it is generally cleaner to return the condition.

Reply:
@hweeks Good point!